### ISSUE TITLE
react 16.0.0-alpha.3 on Stable branch?

### ISSUE THREAD

**Issue by brunocascio** [GeekyAnts/NativeBase#717]
Why do you use an alpha version on your stable branch? 
It's not compatible with previous versions...

**Comment by PEM--** [comment]
Indeed this was creating a mess when JS was bundled resulting. I've reverted to `2.1.0`.

**Comment by irwing-reza** [comment]
React Native 0.43 moved to **"react": "16.0.0-alpha.6"**

Might also be related to [this](https://github.com/facebook/react-native/issues/13314)

**Comment by PEM--** [comment]
Yep, currently, both packages puts youin the same mess. If you're using react-native > 0.42.3 and/or native-base > 2.1.0, you can't compile anymore.

**Comment by brunocascio** [comment]
In order to use the alpha version, you need to run:

`rm -rf node_modules && npm install`

Clean ios/android build folder:

`rm -rf android/build && rm -rf ios/build`

After that, just run as usual:

`react-native run-ios or react-native run-android`

**Comment by StevePotter** [comment]
Why the requirement of React Native 0.43?  It just got released and uses React 16 but all 3rd party libraries are on React 15.  Why not just require RN >= 0.42?  That would fix all the NPM peer dependency issues.

**Comment by robtaireeo** [comment]
I agree with last comment.. moving to 0.43 breaks other libraries like react-navigation

**Comment by csjcode** [comment]
Same problem here... I moved to react-native 0.43, upgraded nativebase  and it broke my project. Instructions above for using the react alpha branch didn't work for me. 

Now, I got my app working again by reverting to 0.42 and reinstall native-base back to 2.0.12 (as later versions seem to require 0.43).

**Comment by bishbashbosh123** [comment]
I've spent a while getting my build stable again too.  I've managed to settle for:
```
    "react": "~15.4.0",
    "react-native": "0.42.3",
    "native-base": "2.1.0",
    "react-navigation": "^1.0.0-beta.7",
```
I previously had: `"native-base": "^2.1.0",`

**Comment by shivrajkumar** [comment]
RN > and < 43 dependency issue fixed with v2.1.3
Works for both RN > 43 and RN < 43
